### PR TITLE
fix: Snackbar with no message on Loging in without tenant identifier

### LINKED ISSUE #1046: Snackbar with no message on Loging in without tenant identifier
**Summary:** 

Empty snackbar on clicking login without a tenant identifier.

**Steps to reproduce:** 

On the login page, enter the following credentials:
Username: mifos
Password: password
Instance URL: staging.openmf.org
Port: --empty--
Tenant Identifier: --empty--

Click on Login.

**Expected behaviour:** 

It should show a snackbar with a message for empty/incorrect tenant identifier.

**Observed behaviour:** 

Shows an empty snackbar. (see picture)

**Device and Android version:** 

OnePlus 5
Android 9.0

**Screenshots:** 
screenshot_20190210-221944

### PR DESCRIPTION
**Pull request by AathmanT**
Fixes #1046

Please Add Screenshots If there are any UI changes.
**Before**
52536757-ba893d00-2d84-11e9-9852-0480e1e61e79

**After**
Screenshot_20190503-120621

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifosng-android/src/main/java/com/mifos/objects/mifoserror/MifosError400.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+
+public class MifosError400 implements Parcelable {
+
+    private String timestamp;
+    private String status;
+    private String error;
+    private String message;
+
+
```
Comment (iadeelzafar) [inline mifosng-android/src/main/java/com/mifos/objects/mifoserror/MifosError400.java]: extra line can be removed

File: mifosng-android/src/main/java/com/mifos/objects/mifoserror/MifosError400.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
+/*
+ * This project is licensed under the open source MPL V2.
+ * See https://github.com/openMF/android-client/blob/master/LICENSE.md
+ */
+
+package com.mifos.objects.mifoserror;
+
+
```
Comment (iadeelzafar) [inline mifosng-android/src/main/java/com/mifos/objects/mifoserror/MifosError400.java]: same here

File: mifosng-android/src/main/java/com/mifos/objects/mifoserror/MifosError400.java [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+import android.os.Parcel;
+import android.os.Parcelable;
+
+
```
Comment (iadeelzafar) [inline mifosng-android/src/main/java/com/mifos/objects/mifoserror/MifosError400.java]: here as well

Comment (AathmanT) [reply mifosng-android/src/main/java/com/mifos/objects/mifoserror/MifosError400.java]: I have removed the extra lines.

File: mifosng-android/src/main/java/com/mifos/objects/mifoserror/MifosError400.java [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    public String getMessage() {
+        return message;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+
```
Comment (iadeelzafar) [inline mifosng-android/src/main/java/com/mifos/objects/mifoserror/MifosError400.java]: 2 extra lines here as well.

**Comment by AathmanT** [comment]
Removed those extra lines.

**Comment by AathmanT** [comment]
Added the screenshots. Thank you.

**Pull request closed without merging**